distro/__init__.py

```diff
@@ -1365,7 +1365,3 @@ def main():
         logger.info("Version: %s", distribution_version)
         distribution_codename = dist.codename()
         logger.info("Codename: %s", distribution_codename)
-
```

Comment:
why is this moved to a separate module? wouldn't it be better to define a "console script" entrypoint?

Reply:
There is a console script. That hasn't changed.

It was moved to a module within a package so that additional "data" files can be installed alongside the package. That data file is the PEP-561 py.typed file.

Reply:
Further, using `__main__.py` allows the user to run `python -m distro` once installed. This was supported with the module version, so I carried over this feature.